hack/olm-registry/hypershift-template.yaml

```diff
@@ -0,0 +1,214 @@
+        - apiVersion: operators.coreos.com/v1alpha1
+          kind: Subscription
+          metadata:
+            name: deployment-validation-operator
+            namespace: openshift-deployment-validation-operator
+          spec:
+            channel: ${CHANNEL}
+            config:
+              env:
+              - name: "NAMESPACE_IGNORE_PATTERN"
+                value: "^(openshift.*|kube-.*|default|dedicated-admin|redhat-.*|acm|addon-dba-operator|codeready-.*|prow)$"
+            name: deployment-validation-operator
+            source: deployment-validation-operator-catalog
+            sourceNamespace: openshift-deployment-validation-operator
+        - apiVersion: networking.k8s.io/v1
+          kind: NetworkPolicy
+          metadata:
+            name: allow-from-openshift-insights
+            namespace: openshift-deployment-validation-operator
+          spec:
+            podSelector: {}
+            ingress:
+              - from:
+                - namespaceSelector:
+                    matchLabels:
+                      name: openshift-insights
+          policyTypes:
+            - Ingress
+        - apiVersion: networking.k8s.io/v1
+          kind: NetworkPolicy
+          metadata:
+            name: allow-from-openshift-olm
+            namespace: openshift-deployment-validation-operator
+          spec:
+            podSelector: {}
+            ingress:
+              - from:
+                - namespaceSelector:
+                    matchLabels:
+                      name: openshift-operator-lifecycle-manager
+          policyTypes:
+            - Ingress
+        - apiVersion: v1
+          kind: Service
+          metadata:
+            name: deployment-validation-operator-metrics
+            namespace: openshift-deployment-validation-operator
+            labels:
+              name: deployment-validation-operator
+          spec:
+            ports:
+            - name: http-metrics
+              port: 8383
+              protocol: TCP
+              targetPort: 8383
+            selector:
+              name: deployment-validation-operator
+        - apiVersion: v1
+          kind: ConfigMap
+          metadata:
+            name: deployment-validation-operator-config
+            namespace: openshift-deployment-validation-operator
+            labels:
+              name: deployment-validation-operator
+          data:
+            deployment-validation-operator-config.yaml: |-
+              checks:
+                # if doNotAutoAddDefaults is true, default checks are not automatically added.
+                doNotAutoAddDefaults: false
+
+                # addAllBuiltIn, if set, adds all built-in checks. This allows users to
+                # explicitly opt-out of checks that are not relevant using Exclude.
+                # Takes precedence over doNotAutoAddDefaults, if both are set.
+                addAllBuiltIn: true
+        - apiVersion: policy.open-cluster-management.io/v1
```

Comment:
All above listed objects should be wrapped in this object.
See:
https://github.com/openshift/hypershift-dataplane-metrics-forwarder/blob/d400ebc5166c0b3f0167973558b11fd7a39487b9/hack/templates/00-hypershift-dataplane-metrics-forwarder.SelectorSyncSet.yaml.tmpl#L123